### PR TITLE
Feat/bu error message file count

### PR DESCRIPTION
**Pull request by joannafawl**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/core/test-utils/MockBulkUploadService.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const ValidatedFile = ValidatedFileBuilder();
+
+@Injectable()
+export class MockBulkUploadService extends BulkUploadService {
+  // public getUploadedFiles(establishmentUid, requiredTiles): Observable<BenchmarksResponse> {
+  //   return of(benchmarksData);
+  // }
+  // public getAllRankingData(establishmentUid): Observable<AllRankingsResponse> {
+  //   return of(allRankingsData);
+  // }
```
Comment (aaron-russell) [inline src/app/core/test-utils/MockBulkUploadService.ts:29]: No point in having commented out code if it's not used

File: src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    expect(component.fixture.componentInstance.preValidationErrorMessage).toEqual('');
+  });
+
+  it('should change the error message if there is only one file', async () => {
+    const { component } = await setup();
+    const dummyFiles = [ValidatedFile];
+    component.fixture.componentInstance.uploadedFiles = dummyFiles;
+    component.fixture.componentInstance.preValidateCheck();
+    component.fixture.detectChanges();
+    // component.debug();
```
Comment (aaron-russell) [inline src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.spec.ts:71]: Same again :)

File: src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.ts [hunk 3]
```diff
@@ -53,6 +53,7 @@ export class DragAndDropFilesListComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.getUploadedFiles();
     this.preValidateFilesSubscription();
+    console.log(this.validationComplete);
```
Comment (aaron-russell) [inline src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.ts:56]: No need for console logs in prod

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.ts (hunk 3 above)
Comment (joannafawl) [reply src/app/features/bulk-upload-v2/drag-and-drop-files-list/drag-and-drop-files-list.component.ts:56]: Thanks for pointing those out, I'll change them now :)

**Review by aaron-russell — COMMENTED** [review]
Looks good for me, do you want to sort the LGTM issue too, please?

**Review by chadtomkiss-madetech — APPROVED** [review]

**Review by aaron-russell — APPROVED** [review]

**Pull request merged**